Assignment11.java

```diff
@@ -0,0 +1,22 @@
+/*
+Using TextFile and a Map<Character,Integer>, create a program that takes the file name as a command line argument and counts the occurrence of all the different characters. Save the results in a text file.
+
+*/
+import java.io.*;
+
+Class Assignment11{
+	public static void main(String [] args)thorws Exception{
+		File file=new File(args[0]);
+ 		FileWriter fw=new FileWriter(file.getPath(),true);
+            	BufferedWriter bw=new BufferedWriter(fw);
```

Comment:
Execute and check the outputs before submitting the Assignments
There is no map created here in this case and without creating you are accessing it